br/pkg/backup/client.go

```diff
@@ -633,6 +633,11 @@ func BuildBackupSchemas(
 			default:
 				if tableInfo.SepAutoInc() {
 					globalAutoID, err = autoIDAccess.IncrementID(tableInfo.Version).Get()
+					// For a nonclustered table with auto_increment column, both auto_increment_id and _tidb_rowid are required.
+					// See also https://github.com/pingcap/tidb/issues/46093
+					if rowID, err1 := autoIDAccess.RowID().Get(); err1 == nil {
```

Comment:
need handle error here

Reply:
print a log?
return the error to the caller seems no suitable, because I'm not 100% sure the rowid meta key exist.
If the rowid does not exist, then the error is expected.

Reply:
Handling of err1 != nil updated. @3pointer